src/main/java/com/github/jcustenborder/kafka/connect/redis/package-info.java

```diff
@@ -13,10 +13,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@Title("Redis")
 @Introduction("The Redis plugin is a collection of connectors that are used to interact with a " +
-    "Redis cluster.")
+        "Redis cluster.")
+@Title("Redis")
+@DocumentationWarning("")
```

Comment:
You don't need this unless you are warning the user about something. Take a look at this [page](https://jcustenborder.github.io/kafka-connect-documentation/projects/kafka-connect-spooldir/index.html) for an example. The warning renders as one of the tiles. Basically the Documentation test will grab metadata from all of the connectors and transformations, and generate RSTs. The build process collects all of this and deploys it to my documentation side when there is a merge to master.

Reply:
Fixed in `2b6b624`.